src/browser/ui/dom/DOMProperty.js

```diff
@@ -141,6 +142,19 @@ var DOMPropertyInjection = {
         propName
       );
     }
+
+    for (var polyfillName in Polyfills) {
+      DOMProperty.isPolyfillName[polyfillName] = true;
+
+      if (DOMAttributeNames.hasOwnProperty(polyfillName)) {
+        var attributeName = DOMAttributeNames[polyfillName];
```

Comment:
attributeName is already defined. See line 269 in the Travis CI test of your PR.